lms/envs/common.py

```diff
@@ -709,7 +709,7 @@ def _add_microsite_dirs_to_default_template_engine(settings):
 # The space is required for space-dependent languages like Arabic and Farsi.
 # However, backward compatibility with Ficus older releases is still maintained (space is still not valid)
 # in the AccountCreationForm and the user_api through the ENABLE_UNICODE_USERNAME feature flag.
-USERNAME_REGEX_PARTIAL = r'[\w .@_+-]+'
+USERNAME_REGEX_PARTIAL = r'[^\/]+'
```

Comment:
so here you accept any character in the world except for `/`, don't you think this is a dangerous assumption?

Reply:
This is what our users were using !